include/net/download_client.h

```diff
@@ -302,28 +303,12 @@ int download_client_file_size_get(struct download_client *client, size_t *size);
  */
 int download_client_downloaded_size_get(struct download_client *client, size_t *size);
 
-/**
- * @brief Initiate disconnection.
- *
- * Request client to disconnect from the server. This does not block.
- * When client have been disconnected, it send @ref DOWNLOAD_CLIENT_EVT_CLOSED event.
- *
- * Request client to disconnect from the server. This does not block.
- * When client has been disconnected, it sends @ref DOWNLOAD_CLIENT_EVT_CLOSED event.
- *
- * @param[in] client	Client instance.
- *
- * @return Zero on success, a negative error code otherwise.
- */
-int download_client_disconnect(struct download_client *client);
-
 /**
  * @brief Download a file asynchronously.
  *
  * This initiates an asynchronous connect-download-disconnect sequence to the target
  * host. When only one file is required from a target server, it can be used instead of
- * separate calls to download_client_set_host(), download_client_start()
- * and download_client_disconnect().
+ * separate calls to download_client_start() and download_client_stop().
  *
  * Downloads are handled one at a time. If previous download is not finished
  * this returns -EALREADY.
```

Comment:
Few lines below, where I cannot add a comment, `CONFIG_DOWNLOAD_CLIENT_HTTP_FRAG_SIZE` is mentioned, this is removed at least somewhere and couldn't quickly find it elsewhere.